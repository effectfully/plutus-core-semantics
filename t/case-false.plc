[[[#case #false] (con 1 ! 1)] (con 1 ! 2)]