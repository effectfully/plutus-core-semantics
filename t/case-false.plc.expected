<generatedTop>
  <k>
    int ( 1 , 2 )
  </k>
  <env>
    .Map
  </env>
  <store>
    0 |-> closure ( .Map , ft , ( lam ff ( fun #unit alpha ) [ ff #unitval ] ) )
    1 |-> int ( 1 , 1 )
    2 |-> int ( 1 , 2 )
    3 |-> closure ( cb |-> 0
    cf |-> 2
    ct |-> 1 , cx , ct )
    4 |-> closure ( cb |-> 0
    cf |-> 2
    ct |-> 1 , cx , cf )
    5 |-> closure ( ff |-> 4
    ft |-> 3 , ux , ux )
  </store>
</generatedTop>
